jboss-kie-db-drivers/configure.sh

```diff
@@ -12,6 +12,8 @@ function link {
 
 link /usr/lib/java/mariadb-java-client.jar $JBOSS_HOME/modules/system/layers/openshift/org/mariadb/main/mariadb-java-client.jar
 link /usr/share/java/postgresql-jdbc.jar $JBOSS_HOME/modules/system/layers/openshift/org/postgresql/main/postgresql-jdbc.jar
+link /usr/share/java/ongres-scram/common.jar $JBOSS_HOME/modules/system/layers/openshift/com/ongres/scram/common/main/ongres-scram-common.jar
```

Comment:
Are we sure that adding the ongres-scram won't add any harm to the non-SSL connections?

Reply:
it shouldn't, but we can assure this later in the QE window.